content/docs/command-reference/exp/apply.md

```diff
@@ -15,21 +15,25 @@ positional arguments:
 ## Description
 
 Restores an `experiment` into the workspace as long as no more Git commits have
-been made after the target experiment (`HEAD` hasn't moved). The `experiment`
-can be referenced by name or hash (see `dvc exp run` for details). This changes
-any files (code, data, <abbr>parameters</abbr>, <abbr>metrics</abbr>, etc.)
-needed to reflect the experiment conditions and results in the workspace.
+been made after the target experiment (`HEAD` hasn't moved). The experiment can
+be referenced by name or hash (see `dvc exp run` for details).
 
-⚠️ Conflicting changes in the workspace are overwritten unless `--no-force` is
-used.
+Specifically, `dvc exp apply` changes any files (code, data,
+<abbr>parameters</abbr>, <abbr>metrics</abbr>, etc.) needed to reflect the
+experiment conditions and results in the workspace.
+
+⚠️ Current changes to the workspace are preserved except if they conflict with
+the experiment in question, in which case they are overwritten (unless
+`--no-force` is used).
 
 This is typically used after choosing a target `experiment` with `dvc exp show`
-or `dvc exp diff`, and before committing it to Git (making it
-[persistent](/doc/user-guide/experiment-management#persistent-experiments)).
+or `dvc exp diff`, and before committing it to Git (making it [persistent].
+
+Note that the history of [checkpoints] found in the `experiment` is **not**
```

Comment:
It might be better to have a ⚠️ here?

Reply:
Idk. This mention of checkpoints is already kind of out of nowhere, especially now that checkpoints info is not in the cmd ref. (updated link to guide). I changed it a bit and made it a note for now.